app/utils/createEntityReducer.js

```diff
@@ -10,36 +10,36 @@ import type { Reducer, AsyncActionType } from 'app/types';
 type EntityReducerOptions = {
   key: string,
   types: {
-    fetch?: ?AsyncActionType | Array<?AsyncActionType>,
+    fetch?: AsyncActionType | Array<AsyncActionType>,
     mutate?: ?AsyncActionType
   },
   mutate?: Reducer,
   initialState?: Object
 };
 
-export function fetching(fetchType?: ?AsyncActionType) {
+export function fetching(fetchTypes?: Array<AsyncActionType>) {
   return (state: any = { fetching: false }, action: any) => {
-    if (!fetchType) {
+    if (!fetchTypes || fetchTypes.length == 0) {
       return state;
     }
 
-    switch (action.type) {
-      case fetchType.BEGIN:
-        return { ...state, fetching: true };
+    for (const fetchType of fetchTypes) {
+      switch (action.type) {
+        case fetchType.BEGIN:
+          return { ...state, fetching: true };
 
-      case fetchType.SUCCESS:
-      case fetchType.FAILURE:
-        return { ...state, fetching: false };
-
-      default:
-        return state;
+        case fetchType.SUCCESS:
+        case fetchType.FAILURE:
+          return { ...state, fetching: false };
+      }
     }
+    return state;
   };
 }
 
 const isNumber = id => !isNaN(Number(id)) && !isNaN(parseInt(id, 10));
 
-export function entities(key: string, fetchType?: ?AsyncActionType) {
+export function entities(key: string, fetchTypes: ?Array<AsyncActionType>) {
```

Comment:
Should not be a [maybe type](https://flow.org/en/docs/types/maybe/), in order to simplify it.

export function entities(key: string, fetchTypes: Array<AsyncActionType>) {

Reply:
We can also accept both a value, and an array. Like we to for `createEntityReducer`.